scripts/test/ipython.sh

```diff
@@ -45,6 +45,7 @@ DATASOURCE="mysql://root:root@tcp(127.0.0.1:3306)/?maxAllowedPacket=0"
 export PYTHONPATH=$GOPATH/src/sqlflow.org/sqlflow/python
 
 sqlflowserver &
+sleep 10
```

Comment:
Is it possible to change this sleep to a while loop that pings the SQLFlow server?